SH4RE/SH4RE/Account/FavouritesModel.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  FavouritesModel.swift
+//  SH4RE
+//
+//  Created by Americo on 2023-03-17.
+//
+
+import SwiftUI
+
+class FavouritesModel: ObservableObject {
+    @Published var favourites:Set<String> = []
```

Comment:
@bbryanb333 this variable should be set by retrieving from favourites table in firebase